mapflow/functional/api/data_catalog_api.py

```diff
@@ -157,6 +165,51 @@ def get_image_preview(self,
                       use_default_error_handler=True
                       )
 
+    def get_image_preview_l(self,
+                            image: ImageReturnSchema,
+                            extent: QgsRectangle,
+                            callback: Callable,
+                            image_id: str = ""):
+        self.http.get(url=image.preview_url_l,
+                      callback=callback,
+                      use_default_error_handler=False,
+                      error_handler=self.image_preview_l_error_handler,
+                      callback_kwargs={"extent": extent,
+                                       "image_id": image_id})
+
+    def display_image_preview(self,
```

Comment:
I don't remember, maybe we discussed it to be like this, but I think that display_image_preview contains some app logic, so it belongs to service, not API